index.html

```diff
@@ -0,0 +1,94 @@
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="https://kit.fontawesome.com/c373984c3f.js" crossorigin="anonymous"></script>
+        <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,900;1,800&family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="style.css">
+        <title>My Portfolio</title>
+    </head>
+
+    <body>
+        <header class="intro">
+
+            <div class="toolbar">
+
+                    <h1 class="toolbar-logo font-family-roboto">Belden Mugauri</h1>
+
+                    <button class="menu-button" id="menu-btn" aria-controls="nav-bar">
+                        <i class="fa-solid fa-bars"><span class="sr-only">Menu</span></i>
+                    </button>
+
+  
+                    <nav id="nav-bar" data-visible="false" class="primary-navigation font-family-roboto"> 
+                        
+
+                        <ul>
+                            <li class="nav-item font-family-roboto">
+                                <a href="#">Portfolio</a>
+                            </li>
+                
+                            <li class="nav-item font-family-roboto">
+                                <a href="#">About</a>
+                            </li>
+                
+                            <li class="nav-item font-family-roboto"> 
+                                <a href="#">Contact</a> 
+                            </li>
+                   </ul>
+
+                    </nav>
+            </div> 
+
+
+
+            <div class="primary-content">
+                <img class="elipse-3" src="images\Ellipse 3.jpg" alt="">
+                <img class="elipse-4" src="images\Ellipse 4.jpg" alt="">
+                <img class="elipse-11" src="images\Ellipse 11.png" alt="">
+                <img class="elipse-12"src="images\Ellipse 12 (1).jpg" alt="">
+                <img class="elipse-13" src="images\Ellipse 12.jpg" alt="">
+                <div>
+                   <div class="primary-information">
+
+                            <h1 class="primary-title font-family-crete">I'm Belden.<br> Glad to see you!</h1>
+                            
+                            <p class="description font-family-poppins">
+                                I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+                            </p>
+
+                            <span class="social-header font-family-poppins">LET'S CONNECT</span>
+                    </div>
```

Comment:
- [ ] Kindly use either a `<button>` or ` <a>` tag for `let's connect` element for semantic reasons